fastapi_sqla/__init__.py

```diff
@@ -63,6 +65,8 @@ def setup(app: FastAPI):
 def startup():
     lowercase_environ = {k.lower(): v for k, v in os.environ.items()}
     engine = engine_from_config(lowercase_environ, prefix="sqlalchemy_")
+    event.listen(engine.engine, "do_connect", set_connection_token)
```

Comment:
This should be in the `fastapi_sqla.aws_rds_iam_support` module as it is irrelevant if rds support is not needed.